app/api/__init__.py

```diff
@@ -120,8 +122,14 @@
 api.route(EventCopyrightRelationship, 'copyright_event',
           '/event-copyright/<int:id>/relationships/event')
 
-
 # tax
 api.route(TaxList, 'tax_list', '/events/<int:id>/tax')
 api.route(TaxDetail, 'tax_detail', '/tax/<int:id>', '/events/<int:event_id>/tax')
 api.route(TaxRelationship, 'tax_event','/tax/<int:id>/relationships/event')
+
+# event invoices
+api.route(EventInvoiceList, 'event_invoice_list', '/event-invoices', '/events/<int:event_id>/event-invoices',
```

Comment:
I don't think `/event-invoices` endpoint is needed since this is to be created as just a relationship of Event. @niranjan94 @magdalenesuo @shubham-padia what do you think?